packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -0,0 +1,497 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'package:flutter/cupertino.dart';
+
+const EdgeInsets _kHorizontalItemPadding =
+    const EdgeInsets.symmetric(horizontal: 16.0, vertical: 0.0);
+
+const Radius _kDefaultCornerRadius = const Radius.circular(3.0);
+
+/// An iOS-style segmented control bar
+///
+/// Displays the provided [children] widgets in a horizontal list.
+/// Exactly one [children] widget is always selected. By default,
```

Comment:
Exactly one [children] widget --> Exactly one widget from the [children] list
Or, you can just change "[children]" to "child" (without the brackets), since you link to it in the previous line already. The text should always read like proper English.